docs/contributing.rst

```diff
@@ -124,14 +124,15 @@ Example: ``./test/libsolidity/syntaxTests/double_stateVariable_declaration.sol``
     // ----
     // DeclarationError: (36-52): Identifier already declared.
 
-A syntax test must contain at least the contract under test itself, followed by the separator ``// ----``. The following comments are used to describe the
-expected compiler errors or warnings. The number range denotes the location in the source where the error occurred.
-In case the contract should compile without any errors or warning, the section after the separator has to be empty
-and the separator can be left out completely.
+A syntax test must contain at least the contract under test itself, followed by the separator ``// ----``.
+You use the comments to describe the expected compiler errors or warnings. The number range denotes the
```

Comment:
I think passive would again be better here, because most of the time, the comments are auto-generated.

Reply:
@chriseth this is a great point, and actually something quite important to mention, because as far as I can see, it's not. But… Are there any more details on this process anywhere? And when you say "most of the time", what do you mean?

Or am I missing something here?

Reply:
It is mentioned at the end of the section, although perhaps not too explicitly. We should probably mention it earlier, but probably better to do that in a new PR.

Reply:
OK, updated based on this and other comments and will create a new PR in the near future.